sql/tests: TestMonotonicInserts/distsql=off failed

sql/tests.TestMonotonicInserts/distsql=off [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3290447&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3290447&tab=artifacts#/) on master @ [5efb168f40b3033fdd7c3a048ae33aecad5d17c9](https://github.com/cockroachdb/cockroach/commits/5efb168f40b3033fdd7c3a048ae33aecad5d17c9):


```
Slow failing tests:
TestMonotonicInserts/distsql=off - 2560.16s

Slow passing tests:
Example_demo - 222.60s
TestMergeQueueSeesNonVoters - 186.01s
TestIndexCleanupAfterAlterFromRegionalByRow - 105.68s
TestLearnerSnapshotFailsRollback - 98.03s
TestBackupRestoreTenant - 89.41s
TestImportAvro - 81.54s
TestRestoreOldVersions - 73.37s
TestMultiNodeExportStmt - 72.15s
TestStoreRangeMergeTimestampCache - 65.05s
TestScatterRandomizeLeases - 57.44s
TestChangefeedSchemaChangeAllowBackfill - 55.62s
TestDefaultColumns - 55.36s
TestBackupRestoreUserDefinedSchemas - 54.78s
TestShowLastQueryStatistics - 54.42s
TestClosedTimestampFrozenAfterSubsumption - 53.79s
TestTruncatedStateMigration - 53.52s
TestInvertedIndex - 51.76s
TestGossipFirstRange - 49.96s
TestReadWriteFile - 48.55s
TestBumpSideTransportClosed - 48.51s
```
<details><summary>Reproduce</summary>
<p>
To reproduce, try:

```bash
make stressrace TESTS=TestMonotonicInserts/distsql=off PKG=./pkg/sql/tests TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/sql-queries petermattis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts/distsql=off.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-9259

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3669139&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3669139&tab=artifacts#/) on master @ [b180b4df799345746356d8fdde649e8764b3ada1](https://github.com/cockroachdb/cockroach/commits/b180b4df799345746356d8fdde649e8764b3ada1):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.35s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on211414969
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=5b6c4927 key=/Table/53/1/707053862749011969/0 pri=0.04728194 epo=0 ts=1635846114.934636023,1 min=1635846114.882010031,0 seq=1} lock=true stat=PENDING rts=1635846114.882010031,0 wto=false gul=1635846115.382010031,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on211414969
    --- FAIL: TestMonotonicInserts/distsql=on (7.03s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3701194&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3701194&tab=artifacts#/) on master @ [5d68dff8e52616ed92798c1a01b370e62f57c6a8](https://github.com/cockroachdb/cockroach/commits/5d68dff8e52616ed92798c1a01b370e62f57c6a8):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (17.04s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on791050756
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_ABORT_SPAN): "sql txn" meta={id=2d89c549 key=/Table/53/1/709043189009842177/0 pri=0.05854541 epo=1 ts=1636453209.147666498,0 min=1636453209.074598591,0 seq=0} lock=true stat=ABORTED rts=1636453209.147666498,0 wto=false gul=1636453209.574598591,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on791050756
    --- FAIL: TestMonotonicInserts/distsql=on (8.54s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3729235&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3729235&tab=artifacts#/) on master @ [13669f9c9bd92a4c3b0378a558d7735f122c4e72](https://github.com/cockroachdb/cockroach/commits/13669f9c9bd92a4c3b0378a558d7735f122c4e72):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (17.43s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off678654207
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=dd7c3efa key=/Table/53/1/710456758128869379/0 pri=0.06083315 epo=0 ts=1636884596.187447065,1 min=1636884596.158583555,0 seq=1} lock=true stat=PENDING rts=1636884596.158583555,0 wto=false gul=1636884596.658583555,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off678654207
    --- FAIL: TestMonotonicInserts/distsql=off (8.73s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3740368&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3740368&tab=artifacts#/) on master @ [e921036c0640d833548363cd8f3fea78ae534bd1](https://github.com/cockroachdb/cockroach/commits/e921036c0640d833548363cd8f3fea78ae534bd1):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (16.88s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off466350889
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=b5eb318c key=/Table/53/1/711027959604543491/0 pri=0.04609898 epo=1 ts=1637058913.066354559,0 min=1637058913.024682252,0 seq=0} lock=true stat=ABORTED rts=1637058913.066354559,0 wto=false gul=1637058913.524682252,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off466350889
    --- FAIL: TestMonotonicInserts/distsql=off (8.34s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3753643&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3753643&tab=artifacts#/) on master @ [1a078671083946342ecf610a6fb899df2d1783d3](https://github.com/cockroachdb/cockroach/commits/1a078671083946342ecf610a6fb899df2d1783d3):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.40s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off3184521766
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=e2018c6e key=/Table/53/1/711582371751919619/0 pri=0.09195921 epo=0 ts=1637228106.211273089,1 min=1637228106.199602628,0 seq=1} lock=true stat=PENDING rts=1637228106.199602628,0 wto=false gul=1637228106.699602628,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off3184521766
    --- FAIL: TestMonotonicInserts/distsql=off (7.11s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3825933&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3825933&tab=artifacts#/) on master @ [c2a0f7384b5ed3030bcfd11bed4e8db7f3fa62f8](https://github.com/cockroachdb/cockroach/commits/c2a0f7384b5ed3030bcfd11bed4e8db7f3fa62f8):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.10s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on1116794460
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=ced1178b key=/Table/56/1/715275766115893251/0 pri=0.06569326 epo=1 ts=1638355240.724700195,1 min=1638355240.689302245,0 seq=1} lock=true stat=PENDING rts=1638355240.690627780,0 wto=false gul=1638355241.189302245,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on1116794460
    --- FAIL: TestMonotonicInserts/distsql=on (7.05s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3927614&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3927614&tab=artifacts#/) on master @ [150caaf45cf8633aca2e5d0ba7ef7bbc2a285efb](https://github.com/cockroachdb/cockroach/commits/150caaf45cf8633aca2e5d0ba7ef7bbc2a285efb):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.43s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=off745533585
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=8b451763 key=/Table/56/1/719520969604661250/0 pri=0.02902821 epo=0 ts=1639650774.022455741,1 min=1639650774.005086398,0 seq=1} lock=true stat=PENDING rts=1639650774.005086398,0 wto=false gul=1639650774.505086398,0
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=off (7.15s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3939029&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3939029&tab=artifacts#/) on master @ [3b4e180a23f5121e0d4106eb3dc5f61ebc314188](https://github.com/cockroachdb/cockroach/commits/3b4e180a23f5121e0d4106eb3dc5f61ebc314188):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.59s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=on2732315841
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=54dd7167 key=/Table/56/1/719779219320274947/0 pri=0.05326685 epo=1 ts=1639729585.585727137,0 min=1639729585.560537932,0 seq=0} lock=true stat=ABORTED rts=1639729585.585727137,0 wto=false gul=1639729586.060537932,0
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=on (7.53s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3988971&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3988971&tab=artifacts#/) on master @ [8d4953ef35bf310c72108c2790ba21971e9e7291](https://github.com/cockroachdb/cockroach/commits/8d4953ef35bf310c72108c2790ba21971e9e7291):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.39s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=off2689842845
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: ReadWithinUncertaintyIntervalError: read at time 1640255103.458125421,0 encountered previous write with future timestamp 1640255103.458125421,1 within uncertainty interval ``t <= (local=1640255103.459902704,0, global=1640255103.958125421,0)``; observed timestamps: [{1 1640255103.459902704,0}]: "sql txn" meta={id=a0a81669 key=/Min pri=0.09473171 epo=0 ts=1640255103.458125421,0 min=1640255103.458125421,0 seq=0} lock=false stat=PENDING rts=1640255103.458125421,0 wto=false gul=1640255103.958125421,0
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=off (7.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4002664&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4002664&tab=artifacts#/) on master @ [f3710ae65e327d0795123029db679f8c9f3e610e](https://github.com/cockroachdb/cockroach/commits/f3710ae65e327d0795123029db679f8c9f3e610e):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.18s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=off1288669556
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=83b5cc55 key=/Table/56/1/722915023327166467/0 pri=0.09015929 epo=1 ts=1640686557.004152609,1 min=1640686556.948280090,0 seq=1} lock=true stat=PENDING rts=1640686556.954139915,0 wto=false gul=1640686557.448280090,0
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=off (7.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4020750&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4020750&tab=artifacts#/) on master @ [3471dc50db8406f3faa638940492dbe0cab53435](https://github.com/cockroachdb/cockroach/commits/3471dc50db8406f3faa638940492dbe0cab53435):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.17s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=on2737322582
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=2b45dbd7 key=/Table/56/1/723764616612937731/0 pri=0.06832215 epo=0 ts=1640945832.307959860,1 min=1640945832.275212856,0 seq=1} lock=true stat=PENDING rts=1640945832.275212856,0 wto=false gul=1640945832.775212856,0
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=on (7.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4024733&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4024733&tab=artifacts#/) on master @ [fa9480a8b661b31a3c5e4c3428595b3e7c320700](https://github.com/cockroachdb/cockroach/commits/fa9480a8b661b31a3c5e4c3428595b3e7c320700):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.62s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off2876878667
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=c25c3c5b key=/Table/56/1/724093549939851266/0 pri=0.06456762 epo=0 ts=1641046214.797092863,1 min=1641046214.778675279,0 seq=1} lock=true stat=PENDING rts=1641046214.778675279,0 wto=false gul=1641046215.278675279,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off2876878667
    --- FAIL: TestMonotonicInserts/distsql=off (7.21s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4030354&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4030354&tab=artifacts#/) on master @ [69e9a90e816bb11268c15d24f278d3ac9cee22ea](https://github.com/cockroachdb/cockroach/commits/69e9a90e816bb11268c15d24f278d3ac9cee22ea):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (22.48s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=on393926620
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=ccafd6b7 key=/Table/56/1/724610551729946627/0 pri=0.05901125 epo=2 ts=1641203991.234105117,0 min=1641203991.137321648,0 seq=0} lock=true stat=ABORTED rts=1641203991.234105117,0 wto=false gul=1641203991.637321648,0
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=on (11.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4029529&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4029529&tab=artifacts#/) on master @ [a9bb445953d3bd80f035fab1c521e8acf0e25c34](https://github.com/cockroachdb/cockroach/commits/a9bb445953d3bd80f035fab1c521e8acf0e25c34):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.09s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off69283027
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=e3263f5c key=/Table/56/1/724656979895681027/0 pri=0.03416501 epo=0 ts=1641218159.948989823,1 min=1641218159.928626097,0 seq=1} lock=true stat=PENDING rts=1641218159.928626097,0 wto=false gul=1641218160.428626097,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off69283027
    --- FAIL: TestMonotonicInserts/distsql=off (6.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4044645&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4044645&tab=artifacts#/) on master @ [fa91c719392104a9d288db44b6afad6deab1ef87](https://github.com/cockroachdb/cockroach/commits/fa91c719392104a9d288db44b6afad6deab1ef87):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.31s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on2565320692
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=6e539f9a key=/Table/56/1/725235852474253314/0 pri=0.05954141 epo=0 ts=1641394817.864595625,1 min=1641394817.817508022,0 seq=1} lock=true stat=PENDING rts=1641394817.817508022,0 wto=false gul=1641394818.317508022,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=on2565320692
    --- FAIL: TestMonotonicInserts/distsql=on (7.18s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4064897&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4064897&tab=artifacts#/) on master @ [f377f02d48f6acabcd71d1d6c9dadbd5e9eed553](https://github.com/cockroachdb/cockroach/commits/f377f02d48f6acabcd71d1d6c9dadbd5e9eed553):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.59s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off841897831
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=e91a0f2b key=/Table/56/1/725804024965693443/0 pri=0.03577219 epo=0 ts=1641568210.350059288,1 min=1641568210.333645973,0 seq=1} lock=true stat=PENDING rts=1641568210.333645973,0 wto=false gul=1641568210.833645973,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off841897831
    --- FAIL: TestMonotonicInserts/distsql=off (7.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4073070&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4073070&tab=artifacts#/) on master @ [d66c14663e027a5d9699b8a15e3d6f5821877ef5](https://github.com/cockroachdb/cockroach/commits/d66c14663e027a5d9699b8a15e3d6f5821877ef5):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.53s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off1144542639
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=3fffc4cc key=/Table/56/1/726082894678982659/0 pri=0.04725405 epo=0 ts=1641653314.632620309,1 min=1641653314.596070756,0 seq=1} lock=true stat=PENDING rts=1641653314.596070756,0 wto=false gul=1641653315.096070756,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off1144542639
    --- FAIL: TestMonotonicInserts/distsql=off (7.21s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4078221&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4078221&tab=artifacts#/) on master @ [6ff6c77e7aebec2edc947c1ed99a1f7cb67e1a3c](https://github.com/cockroachdb/cockroach/commits/6ff6c77e7aebec2edc947c1ed99a1f7cb67e1a3c):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.06s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off1756937345
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=0d020269 key=/Table/56/1/726363223291396097/0 pri=0.03165435 epo=1 ts=1641738864.150014343,0 min=1641738864.117164034,0 seq=0} lock=true stat=ABORTED rts=1641738864.150014343,0 wto=false gul=1641738864.617164034,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off1756937345
    --- FAIL: TestMonotonicInserts/distsql=off (7.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4117538&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4117538&tab=artifacts#/) on master @ [20eaf0b415f1df361246804e5d1d80c7a20a8eb6](https://github.com/cockroachdb/cockroach/commits/20eaf0b415f1df361246804e5d1d80c7a20a8eb6):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.26s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off3075223975
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=82f1c7c5 key=/Table/56/1/727736147716800515/0 pri=0.07153610 epo=1 ts=1642157847.424064717,0 min=1642157847.399905401,0 seq=0} lock=true stat=ABORTED rts=1642157847.424064717,0 wto=false gul=1642157847.899905401,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off3075223975
    --- FAIL: TestMonotonicInserts/distsql=off (7.15s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4128466&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4128466&tab=artifacts#/) on master @ [78419450178335b31f542bd1b14fefdf4ecee0e8](https://github.com/cockroachdb/cockroach/commits/78419450178335b31f542bd1b14fefdf4ecee0e8):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.42s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off3518742840
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_ABORT_SPAN): "sql txn" meta={id=e2f9e1f8 key=/Table/56/1/728016141941997570/0 pri=0.03177030 epo=1 ts=1642243294.880915922,0 min=1642243294.849771694,0 seq=0} lock=true stat=ABORTED rts=1642243294.880915922,0 wto=false gul=1642243295.349771694,0
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off3518742840
    --- FAIL: TestMonotonicInserts/distsql=off (7.13s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4228240&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4228240&tab=artifacts#/) on master @ [ba5c8326d4acf773dce97a1f4eff735b566a2522](https://github.com/cockroachdb/cockroach/commits/ba5c8326d4acf773dce97a1f4eff735b566a2522):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.24s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=on3891828557
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: retrying txn failed after 50 attempts. original error: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=75249799 key=/Table/56/1/731470606993293315/0 pri=0.07092557 epo=2 ts=1643297513.951580417,0 min=1643297513.900674538,0 seq=0} lock=true stat=ABORTED rts=1643297513.951580417,0 wto=false gul=1643297514.400674538,0.
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=on (7.25s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4248261&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4248261&tab=artifacts#/) on master @ [75ba483e01a04e6ad707b3f0c00d645b6c92cb52](https://github.com/cockroachdb/cockroach/commits/75ba483e01a04e6ad707b3f0c00d645b6c92cb52):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.87s)
=== RUN   TestMonotonicInserts/distsql=on
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=on1412137651
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: retrying txn failed after 50 attempts. original error: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=dcd7e429 key=/Table/56/1/731980932428136450/0 pri=0.04857611 epo=1 ts=1643453252.916556280,0 min=1643453252.886565039,0 seq=0} lock=true stat=ABORTED rts=1643453252.916556280,0 wto=false gul=1643453253.386565039,0.
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=on (7.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4278401&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4278401&tab=artifacts#/) on master @ [0e5eb08368020e7633142fd60f0f818fcd46510d](https://github.com/cockroachdb/cockroach/commits/0e5eb08368020e7633142fd60f0f818fcd46510d):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.38s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off1365394861
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: retrying txn failed after 50 attempts. original error: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=ac4bd417 key=/Table/56/1/733142542322827267/0 pri=0.06458736 epo=1 ts=1643807748.123023473,0 min=1643807748.098908480,0 seq=0} lock=true stat=ABORTED rts=1643807748.123023473,0 wto=false gul=1643807748.598908480,0.
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off1365394861
    --- FAIL: TestMonotonicInserts/distsql=off (7.14s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4290960&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4290960&tab=artifacts#/) on master @ [61e2304e5c0fc448942a8659af129851b34a00f9](https://github.com/cockroachdb/cockroach/commits/61e2304e5c0fc448942a8659af129851b34a00f9):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.71s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off429211306
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: retrying txn failed after 50 attempts. original error: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionAbortedError(ABORT_REASON_PUSHER_ABORTED): "sql txn" meta={id=27a826a0 key=/Table/56/1/733436000607272962/0 pri=0.05334131 epo=2 ts=1643897304.487759161,0 min=1643897304.436159182,0 seq=0} lock=true stat=ABORTED rts=1643897304.487759161,0 wto=false gul=1643897304.936159182,0.
    monotonic_insert_test.go:236: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestMonotonicInserts_distsql=off429211306
    --- FAIL: TestMonotonicInserts/distsql=off (7.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestMonotonicInserts [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4326973&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4326973&tab=artifacts#/) on master @ [03c50b0dc6eb45f79ebb7e4ee677f1dafad69565](https://github.com/cockroachdb/cockroach/commits/03c50b0dc6eb45f79ebb7e4ee677f1dafad69565):


```
=== RUN   TestMonotonicInserts
--- FAIL: TestMonotonicInserts (14.40s)
=== RUN   TestMonotonicInserts/distsql=off
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/b0a4fc4c280ad60b26568d64fcb64d2b/logTestMonotonicInserts_distsql=off245427405
    test_log_scope.go:80: use -show-logs to present logs inline
    monotonic_insert_test.go:183: *crdb.MaxRetriesExceededError: retrying txn failed after 50 attempts. original error: pq: restart transaction: TransactionRetryWithProtoRefreshError: TransactionRetryError: retry txn (RETRY_SERIALIZABLE - failed preemptive refresh): "sql txn" meta={id=8434d46f key=/Table/106/1/734812790089580547/0 pri=0.06186985 epo=0 ts=1644317467.281665236,1 min=1644317467.278843134,0 seq=1} lock=true stat=PENDING rts=1644317467.278843134,0 wto=false gul=1644317467.778843134,0.
    monotonic_insert_test.go:236: -- test log scope end --
    --- FAIL: TestMonotonicInserts/distsql=off (7.11s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMonotonicInserts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/tests/monotonic_insert_test.go
+++ b/pkg/sql/tests/monotonic_insert_test.go
@@ -77,7 +77,6 @@ type mtClient struct {
 //   https://github.com/jepsen-io/jepsen/blob/master/cockroachdb/src/jepsen/cockroach/monotonic.clj
 func TestMonotonicInserts(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	skip.WithIssue(t, 68784, "flaky test")
 
 	for _, distSQLMode := range []sessiondatapb.DistSQLExecMode{
 		sessiondatapb.DistSQLOff, sessiondatapb.DistSQLOn,
@@ -93,6 +92,8 @@ func testMonotonicInserts(t *testing.T, distSQLMode sessiondatapb.DistSQLExecMod
 	defer log.Scope(t).Close(t)
 
 	skip.UnderShort(t)
+	skip.UnderRace(t) // Too slow under race.
+
 	ctx := context.Background()
 	tc := testcluster.StartTestCluster(
 		t, 3,

